zebra/rt_netlink.c

```diff
@@ -1381,8 +1381,10 @@ static int netlink_route_change_read_multicast(struct nlmsghdr *h,
 				*(struct in6_addr *)RTA_DATA(tb[RTA_DST]);
 	}
 
-	if (tb[RTA_EXPIRES])
-		m->lastused = *(unsigned long long *)RTA_DATA(tb[RTA_EXPIRES]);
+	if (tb[RTA_EXPIRES]) {
+		uint32_t temporary = *(uint32_t *)RTA_DATA(tb[RTA_EXPIRES]);
```

Comment:
I'm a little unsure about this - it looks like iproute2 does use a 64-bit copy for this attr?

Reply:
yeah fair enough.  I fixed it to follow the same logic as iproute2 here.